controllers/routes.js

```diff
@@ -0,0 +1,92 @@
+//This file is for handling the routing of urls
+
+var express = require('express')
+var router = express.Router()
+var request = require('request')
+var config = require('../config/config')
+var path = require('path')
+
+var appDir = path.dirname(require.main.filename)
+
+router.get('/', function(req, res) {
+  res.sendFile(appDir+'/views/index.html')
+})
+
+//Api to fetch profile of a particular file
+router.get('/profile/:id',function(req,res) {
+    apiUrl = 'https://api.practo.com/doctors/'+req.params.id+'?with_relations=true'
+
+    console.log('check1')
+    request.get(
+    {
+        url : apiUrl,
+        headers : 
+        {
+            'X-CLIENT-ID' : config.clientId,
+            'X-API-KEY' : config.token
+        }
+    },function(error,response,body){
+        if(!error && response.statusCode == 200) 
+        {
```

Comment:
bhai, parenthesis should be placed at the same line 
`if(!error && response.statusCode == 200) {`